exchange/paperwallet.go

```diff
@@ -586,7 +630,21 @@ func (p *PaperWallet) CreateOrderMarket(side model.SideType, pair string, size f
 	p.Lock()
 	defer p.Unlock()
 
-	return p.createOrderMarket(side, pair, size)
+	// I assume that the CreateOrderMarket method is used only for SPOT market.
+	// The future market (with leverage) will use the quote method. Therefore, in the current case we'll call
+	// the createOrderMarket with leverage 1
+	return p.createOrderMarket(side, pair, size, p.GetPositionLeverage(pair, size))
+}
+
+func (p *PaperWallet) GetPositionLeverage(pair string, size float64) int {
+	// TODO: Find a better way to get the position leverage
+	for _, order := range p.orders {
+		if order.Pair == pair && order.Quantity == size {
+			return order.Leverage
+		}
+	}
+
+	return 1.0
```

Comment:
Maybe we can include this parameter inside the `Position` method, it works for you?

Reply:
Indeed that's the best solution. However, I added this on order because the order is used in the chart. If we can find another way to draw the chart using both positions and order, will be really awesome.